File: test/source/cond/impl/cond.pas
var
	a: Integer;
begin
	ReadLn(a);
	if (a = 0) or (a = 1) then
		WriteLn('Nice choice!')
	else if a < 0 then
		WriteLn('You went negative')
	else
		WriteLn('WOW SUCH BIG NUMBER');
end.

File: test/dest/cond.hs
import Control.Monad
import Control.Applicative
main = do
           a <- ((read) <$> (getLine)) :: IO Int
           () <- if ((a) == (0)) || ((a) == (1))
                 then do
                          (putStrLn) ("Nice choice!")
                          (return) (())
                 else do
                          () <- if (a) < (0)
                                then do
                                         (putStrLn) ("You went negative")
                                         (return) (())
                                else do
                                         (putStrLn) ("WOW SUCH BIG NUMBER")
                                         (return) (())
                          (return) (())
           (return) (())